### ISSUE TITLE
When building "in" queries, third and subsequent values are indexed "2" instead of incrementing

### ISSUE THREAD

**Issue by jazmatician** [rappen/FetchXMLBuilder#238]
When building a FetchXml with an "in" clause, like the following:
```
<fetch top='50' >
  <entity name='sdkmessageprocessingstep' >
    <all-attributes/>
    <filter>
      <condition attribute='stage' operator='in' >
        <value>20</value>
        <value>40</value>
        <value>10</value>
      </condition>
```
The "FetchData" object incorrectly indexes the third (and beyond) items. It generates this:

```
var fetchData = {
		stage: "20",
		stage2: "40",
		stage2: "10",
	};
```
I tested with additional values in the "in" clause, and they were all indexed as "stage2". The keys should all be unique, i.e. `stage3, stage4,` etc.

**Comment by jazmatician** [comment]
This happens in both the C# and JavaScript generators, so I think it's in the GetFetchData method in the base class, but I can't for the life of me see why it would do this. I think this is resolved in #223 but somehow I don't have the latest updates? 
Ah, I see the problem, I need to update to the latest version of XmlToolbox to get this fix. Done, thanks.

**Comment by rappen** [comment]
I just love self-resolving issues! 😁
Glad you got it working 👍
